pyproject.toml

```diff
@@ -55,14 +55,17 @@ sympy = "^1.10.1"
 scikit-learn = "1.0.2"
 pydot = "^1.4.2"
 joblib = "^1.1.0"
-# causalml is wired to use llvmlite 0.36
```

Comment:
Is there a reason these lines related to causalml have changed together with the update to the mistune version?

Reply:
Those dependencies didn't change, but I grouped them together so it was clear that they were related.